src/inverseRobot.js

```diff
@@ -24,7 +24,20 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+
+  for (const key in robot) {
+    const keys = robot[key];
```

Comment:
why `keys`? it's a value from base robot, so name it properly, and it shouldn't be plural it's not an array